FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,15 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImagesMapper.map(data, from: response))
+			default:
```

Comment:
It can be confusing to see that a default case completes with a connectivity error. Also the developer has to keep in mind that `default` actually means `failure`.
There's no extra value in having `default` here and it makes the code harder to understand. 
Please replace `default` with `.failure`
```suggestion
			.failure:
```